internal/redteam/redteam_r3_test.go

```
@@ -0,0 +1,250 @@
+// 管道符/尖括号/RTL/零宽等夹缝字符不得通过注册校验。
+func TestRT2_UserIDCharsetInjectionSurface(t *testing.T) {
+	rejected := []string{
+		"user:admin",   // 冒号（#43/#53/#58）
+		"evil mirror",  // 空格（#51）
+		"mallory:corp", // 夹缝 ID（#58）
+		"user###2",     // 井号（### Pair 块头伪造面）
+		"user[0]",      // 方括号（JSON 结构面）
+		"user`code`",   // 反引号（markdown 围栏面）
+		"user|pipe",    // 管道符（|text_block 渲染面）
+		"<script>",     // 尖括号（HTML/prompt injection 面）
+		"user​zwsp",    // 零宽空格（视觉隐藏）
```

golangci-lint (2.12.2)

[error] ST1018: string literal contains the Unicode format character U+200B, consider using the '\u200b' escape sequence instead

(staticcheck)